omniledger/service/service.go

```diff
@@ -800,25 +808,19 @@ func (s *Service) registerContract(contractID string, c OmniLedgerContract) erro
 // Tries to load the configuration and updates the data in the service
 // if it finds a valid config-file.
 func (s *Service) tryLoad() error {
-	s.storage = &storage{
-		// Set the default to be the same as the default in
-		// skipchain/struct.go, which is 15 seconds.
-		PropTimeout: 15 * time.Second,
-	}
+	s.SetPropagationTimeout(120 * time.Second)
```

Comment:
I guess this is needed because we set the BFT timeout in `SetPropagationTimeout`. This will significantly slow down message propagation when there is a failure. My suggestion is that we keep it as is and set the two timeouts independently. There should be a relationship between the two timeouts but we don't exactly know what it is at the moment.